packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -580,6 +580,32 @@ class FuchsiaDevice extends Device {
   /// [true] if the current host address is IPv6.
   bool get ipv6 => _ipv6 ??= isIPv6Address(id);
 
+  /// Return the address that the device should use to communicate with the
+  /// host.
+  Future<String> get hostAddress async {
+    if (_cachedHostAddress != null) {
+      return _cachedHostAddress;
+    }
+    final RunResult result = await shell('echo \$SSH_CONNECTION');
+    final void Function() fail = () {
+      throwToolExit('Failed to get local address, aborting.\n$result');
+    };
```

Comment:
```suggestion
    }
```

Reply:
Semi-colon was required as it's a local variable declaration, but I changed it to a local function instead to avoid confusion.